specs/commandline/key.md

```diff
@@ -115,10 +115,27 @@ notation key list
 
 Upon successful execution, a list of keys is printed out with information of name, key path, certificate path, key id and plugin name. The default signing key name is preceded by an asterisk. The key id and plugin name are used together to provide the information of the key identifier for the remote key and the plugin associated with it.
 
-### Delete two keys from signing key list
+### Remove a specified key from signing key list
 
 ```shell
-notation key delete <key_name_1> <key_name_2>
+notation key remove <key_name>
 ```
 
-Upon successful execution, the names of deleted signing keys are printed out. Please be noted if default signing key is deleted, Notation will not automatically assign a new default signing key. User needs to update the default signing key explicitly.
+- Upon successful execution, the output message is printed out as below when removed a key created by notation for local testing purpose:
+
+```text
+Removed <key_name> from signing key list. You still need to delete the key file from <key_path> and delete the certificate from <cert_path>.
```

Comment:
```suggestion
Removed <key_name> from Notation signing key list. The source key file is not deleted.
```

Reply:
@FeynmanZhou I thought we agreed on this 2 weeks ago: https://github.com/notaryproject/notation/pull/606#discussion_r1154298416.

Reply:
@patrickzheng200 It's fine that we do not print the `<key_path>` in the output for this release. Currently, there is a [guide](https://github.com/notaryproject/notaryproject.dev/pull/164/files#diff-ebf28445e02ae412163e2fde43e76bee51e880abab8449ac8b4b863fec8d4354R29 ) to tell users to clean up those source key and cert files. We can determine if we should design a feature like `notation generate-test` or support local signing key scenario.